### PR TITLE
[pfs] Show promptflow service info

### PR DESCRIPTION
**Pull request by lalala123123**
# Description
- pfs -h
image

- pfs start
image

- pfs show-status
image
  service not started:
image
  service started:
image

Please add an informative description that covers that changes made by the pull request and link all relevant issues.

# All Promptflow Contribution checklist:
- [ ] **The pull request does not introduce [breaking changes].**
- [ ] **CHANGELOG is updated for new features, bug fixes or other significant changes.**
- [ ] **I have read the [contribution guidelines](../CONTRIBUTING.md).**
- [ ] **Create an issue and link to the pull request to get dedicated review from promptflow team. Learn more: [suggested workflow](../CONTRIBUTING.md#suggested-workflow).**

## General Guidelines and Best Practices
- [ ] Title of the pull request is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).

### Testing Guidelines
- [ ] Pull request includes test coverage for the included changes.

### CODE REVIEW

File: src/promptflow/promptflow/_sdk/_service/entry.py [hunk 1]
```diff
@@ -3,75 +3,117 @@
...
         prog="pfs",
         formatter_class=argparse.RawDescriptionHelpFormatter,
         description="Prompt Flow Service",
     )
 
-    parser.add_argument("-p", "--port", type=int, help="port of the promptflow service")
     parser.add_argument(
         "-v", "--version", dest="version", action="store_true", help="show current PromptflowService version and exit"
     )
+    subparsers = parser.add_subparsers()
```
Comment (wangchao1230) [inline src/promptflow/promptflow/_sdk/_service/entry.py:97]: do we have telemetry for this kind of command?

File: src/promptflow/promptflow/_sdk/_service/utils/utils.py [hunk 2]
```diff
@@ -36,6 +55,35 @@ def get_random_port():
...
+        proc.kill()
+
+
+def get_started_service_info(port):
+    service_info = {}
+    proc = _get_process_by_port(port)
+    if proc:
+        create_time = proc.info["create_time"]
+        process_uptime = datetime.now() - datetime.fromtimestamp(create_time)
+        service_info["uptime"] = str(process_uptime)
```
Comment (wangchao1230) [inline src/promptflow/promptflow/_sdk/_service/utils/utils.py:82]: I was thinking this info is get back from the /live api, but it's ok to be this in this PR

File: src/promptflow/promptflow/_sdk/_service/utils/utils.py [hunk 3]
```diff
@@ -36,6 +55,35 @@ def get_random_port():
...
+        else:
+            for connection in connections:
+                if connection.laddr.port == port:
+                    return proc
+
+
+def kill_exist_service(port):
+    proc = _get_process_by_port(port)
+    if proc:
+        proc.kill()
```
Comment (wangchao1230) [inline src/promptflow/promptflow/_sdk/_service/utils/utils.py:73]: what's the signal send to process here?

Comment (lalala123123) [reply src/promptflow/promptflow/_sdk/_service/utils/utils.py:71]: it send SIGKILL to process in POSIX. In Windows, it will call function in C extension

Comment (wangchao1230) [reply src/promptflow/promptflow/_sdk/_service/utils/utils.py:71]: Maybe we need to do additiaonl sigterm if SIGKILL not gracefully shutdown this guy

Comment (lalala123123) [reply src/promptflow/promptflow/_sdk/_service/utils/utils.py]: change to terminate and wait for the process to be terminated.

File: src/promptflow/promptflow/_sdk/_service/entry.py (hunk 1 above)
Comment (lalala123123) [reply src/promptflow/promptflow/_sdk/_service/entry.py:97]: add telemetry for this command

**Review by wangchao1230 — APPROVED** [review]

**Review by YingChen1996 — APPROVED** [review]

**Review by wangchao1230 — APPROVED** [review]

**Review by wangchao1230 — APPROVED** [review]

**Review by wangchao1230 — APPROVED** [review]

**Review by YingChen1996 — APPROVED** [review]

**Pull request merged**